### PR TITLE
asset search shows video preview

### LINKED ISSUE #6841: Asset Preview Not Visible In Admin Asset Search
# Bug Report

### Expected behavior
Admin asset search should show video previews
### Actual behavior
Admin asset search only shows previews for image types
### Steps to reproduce
Search for a video asset in the admin

# Feature Request

## Feature description
I have found a solution for this and will be submitting a PR

### PR DESCRIPTION
**Pull request by jeremygnoonan**
## Changes in this pull request  
Resolves #6841  

## Additional info  
Simple fix to javascript to pull in video preview thumbnails

### CODE REVIEW

**Comment by dvesh3** [comment]
Hi @jeremygnoonan the fix looks good 😉 but we need to rebase to 6.6 branch. please see new [guidelines](https://github.com/pimcore/pimcore/blob/master/CONTRIBUTING.md#contributing-pull-requests). thanks!

**Comment by dvesh3** [comment]
ping @jeremygnoonan

File: bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js [hunk 1]
```diff
@@ -198,8 +198,14 @@ pimcore.element.selector.asset = Class.create(pimcore.element.selector.abstract,
                 {text: t("filename"), width: 200, sortable: false, dataIndex: 'filename', hidden: true, renderer: Ext.util.Format.htmlEncode},
                 {text: t("preview"), width: 150, sortable: false, dataIndex: 'subtype',
                     renderer: function (value, metaData, record, rowIndex, colIndex, store) {
-                        if(record.data.subtype == "image") {
-                            var route = 'pimcore_admin_asset_getimagethumbnail';
+                        if(record.data.subtype == "image" || record.data.subtype == "video") {
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js:201]: please fix it for documents as well. thanks!
https://github.com/pimcore/pimcore/blob/ea2f5884ec79540abd9093bb6b1f7c0832cfc010/bundles/AdminBundle/Controller/Admin/AssetController.php#L805-L806

**Comment by brusch** [comment]
@jeremygnoonan target branch `master` is fine, as we're classifying it as an improvement. 
But still, please have a look at @dvesh3 feedback. Thanks!

File: bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js (hunk 1 above)
Comment (jeremygnoonan) [reply bundles/AdminBundle/Resources/public/js/pimcore/element/selector/asset.js]: New code should support documents as well

**Comment by jeremygnoonan** [comment]
hi @dvesh3 and @brusch 
I've made the requested changes. Documents, images and videos supported!
Capture

**Comment by jeremygnoonan** [comment]
Also worth noting that document thumbnails are not generated until maintenance runs, so may not be visible right away. i had to manually run maintenance to see changes in real-time.

**Comment by dvesh3** [comment]
@jeremygnoonan thanks a lot! 🙏

**Comment by jeremygnoonan** [comment]
Happy to help !

**Pull request merged**